main.go

```diff
@@ -68,26 +73,50 @@ func hello(c echo.Context) error {
 	return c.String(http.StatusOK, "Hello, World!")
 }
 
-// user
-// @Summary Add a new User
-// @Description Adds a new user to the list of Users
-// @ID add-user
+// OF Trace
+// @Summary Send trace request to OpenFisca
+// @ID of-trace
 // @Accept  json
 // @Produce  json
-// @Success 200 {string} string	"Returns new user"
-// @Router /user [post]
-func user(c echo.Context) error {
-	// Create a new user
-	u := new(User)
-	// Bind the request body to the new user
-	if err := c.Bind(u); err != nil {
-		return err
+// @Success 200 {string} string	"Returns OpenFisca trace response"
+// @Router /trace [post]
+func trace(c echo.Context) (err error) {
+
+	data := c.Request().Body
+
+	body, err := ioutil.ReadAll(data)
+	if err != nil {
+		log.Printf("Error reading body: %v", err)
+		// http.Error("can't read body", http.StatusBadRequest)
+		return
+	}
+
+	req, err := http.NewRequest("POST", "http://localhost:5000/trace", bytes.NewBuffer(body))
+	if err != nil {
+		fmt.Println("Error in req: ", err)
+	}
+	req.Header.Add("Content-Type", "application/json")
+
+	// Create a Client
+	client := &http.Client{}
+
+	// Do sends an HTTP request and
+	resp, err := client.Do(req)
+	if err != nil {
+		fmt.Println("error in request: ", err)
 	}
 
-	u.UserID = incUserCount()
+	// Defer the closing of the body
+	defer resp.Body.Close()
+
+	body, err = ioutil.ReadAll(resp.Body)
+	if err != nil {
+		//log.Printf("Error reading body: %v", err)
+		//http.Error(w, "can't read body", http.StatusBadRequest)
+		return
+	}
 
-	//TODO: Do something with new User
+	json := json.RawMessage(body)
```

Comment:
Is this step necessary since were not using the body? You should try sending the body directly back if thats the intended behaviour